src/components/Search.js

```diff
@@ -28,14 +30,58 @@ const Search = props => (
                 />
               </td>
               <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
+                <Button label="Ids" onClick={()=>{console.log("Id button")}} />
               </td>
+
+
+
               {/* Add search by name here */}
+
             </tr>
           </tbody>
         </table>
       </div>
     </div>
+    <div className="row">
+      <div className="col">
+        <table className="table search-table">
+          <thead>
+            <tr>
+              <th>Search by name</th>
+              <th />
+              <th></th>
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <input
```

Comment:
The input fields for searchs are very similar, its not part of the exercise but could be move into a separate component as well.